### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ColinMaudry**
This time I paid attention to reuse the vocabulary and phraseology of the legislation :wink:

### CODE REVIEW

**Comment by ColinMaudry** [comment]
Looking the legislation, I wonder why we called it `rewardsDetails`. The term "reward" is not used in the legislation. Its equivalent in F12, `DETAILS_PAYMENT`, seems to describe (from the data I saw) in what conditions the prizes described in `NUMBER_VALUE_PRIZE` are paid to the winner and/or the participants.

In the legislation, when [defining the information going in the form](https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1585836130257&uri=CELEX:32014L0024#L_2014094EN.01021001-g-008-0008), it gives this definition of `DETAILS_PAYMENT`:

> Payments to be made to all participants, if any.

And this definition of `NUMBER_VALUE_PRIZE`:

> Number and value of any prizes.

Do we rename `rewardsDetails` into `prizePaymentDetails`?

**Comment by ColinMaudry** [comment]
> Looking the legislation, I wonder why we called it `rewardDetails`. The term of reward is not used in the legislation. Its equivalent in F12, `DETAILS_PAYMENT`, seems to describe (from the data I saw) in what conditions the prizes described in `NUMBER_VALUE_PRIZE` are paid to the winner and/or the participants.

**Comment by jpmckinney** [comment]
The form has "V.3 Rewards and jury" so we followed that naming. I assume the form uses "rewards" to mean both "prizes" and "payments" which are different things in the directive. eForms has BT-45 Rewards Other. So, let's leave it as-is.

In https://github.com/open-contracting/european-union-support/pull/23#issuecomment-536020555, we discussed:

> Rename paymentDetails to rewardsDetails … because this aligns most closely to BT-45 (Rewards Other), where the payment is a type of reward, e.g. a payment that is made to all participants, just for participating.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
-# ocds_subcontracting_extension
-Adds a fields for information on subcontracting
+# Design contest
+
+Adds an object in the Tender object to describe a design contest.
```
Comment (jpmckinney) [inline README.md]: … object to the tender …

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+
+## Legal context
+
+In the European Union, this extension's fields correspond to [eForms BG-704 (Reward and jury)](https://github.com/eForms/eForms) and [Title III, Chapter II of Directive 2014/24/EU](https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1585836130257&uri=CELEX:32014L0024#d1e6612-65-1). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
```
Comment (jpmckinney) [inline README.md]: Capitalize Jury.

File: extension.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": {
+    "en": "Design contest"
+  },
+  "description": {
+    "en": "Adds an object in the Tender object to describe a design contest."
```
Comment (jpmckinney) [inline extension.json]: Copy change to readme.

File: release-schema.json [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      }
+    },
+    "DesignContest": {
+      "title": "Design contest",
+      "description": "Information about the organization of a design contest.",
+      "type": "object",
+      "properties": {
+        "selectedParticipants": {
+          "title": "Selected participants",
+          "description": "The parties that have been selected to participate to the design contest.",
```
Comment (jpmckinney) [inline release-schema.json]: have been selected -> have already been selected

That's language from the form. It helps to distinguish those selected in advance from those selected in the course of the contest (depending on how the contest is organized).

File: release-schema.json [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+          "title": "Has prizes",
+          "description": "Whether prizes will be awarded in the design contest.",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "prizes": {
+          "title": "Prizes",
+          "description": "Information about the prizes awarded in the design contest",
```
Comment (jpmckinney) [inline release-schema.json]: Add period to end of description. Same for prizes.description, JuryMember, JuryMember.name.

File: release-schema.json [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+        "rewardsDetails": {
+          "title": "Rewards details",
+          "description": "The details and conditions of the payments to be made to the participants.",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "followUpContracts": {
+          "title": "Follow up contracts",
```
Comment (jpmckinney) [inline release-schema.json]: Add hyphen: "Follow-up"

File: release-schema.json [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+          "description": "Whether any contracts following the design contest will or will not be awarded to the winner or winners of the design contest.",
```
Comment (jpmckinney) [inline release-schema.json]: Can remove "or will not" since that is the meaning of "whether".

File: release-schema.json [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "isBindingJuryDecision": {
+          "title": "Is binding jury decision",
```
Comment (jpmckinney) [inline release-schema.json]: Remove "Is". I don't think we have predicates for field titles elsewhere.

File: release-schema.json [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+          "description": "Whether the jury’s decision is binding on the contracting authority.",
```
Comment (jpmckinney) [inline release-schema.json]: Should we use "procuring entity" instead of "contracting authority"?

**Review by jpmckinney — COMMENTED** [review]
Looks good! Just a few changes.

File: release-schema.json (hunk 9 above)
Comment (ColinMaudry) [reply release-schema.json]: That's what we usually use indeed. Now, we also know that several distinct public entities may be involved in a procedure: the procuring entity (managing the procurement procedure), the payee... The contracting authority could differ from the others by being the only one contractually bound by the resulting contract.

This being said, I'm pretty sure we have used "procuring entity" when meaning "the entity that is contractually bound" in other extensions, we consequently can stick to "procuring entity" for sake of consistency.

File: release-schema.json (hunk 8 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

File: release-schema.json (hunk 7 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

File: release-schema.json (hunk 6 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

File: release-schema.json (hunk 5 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

File: release-schema.json (hunk 4 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

File: extension.json (hunk 3 above)
Comment (ColinMaudry) [reply extension.json]: Done

File: README.md (hunk 2 above)
Comment (ColinMaudry) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (ColinMaudry) [reply README.md]: Done

**Pull request merged**